src/components/ChatForm/ToolUseSwitch.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex, SegmentedControl, Text } from "@radix-ui/themes";
+import { ToolUse } from "../../features/Chat";
+
+type ToolUseSwitchProps = {
+  toolUse: ToolUse;
+  setToolUse: (toolUse: ToolUse) => void;
+};
+
+export const ToolUseSwitch = ({ toolUse, setToolUse }: ToolUseSwitchProps) => {
+  return (
+    <Flex direction="column" gap="8px" align="start">
+      <Text size="2">How fast you want the answer:</Text>
+      <SegmentedControl.Root
```

Comment:
maybe add size="1" to the SegmentedControl.Root, we'll need to see how it looks when in an ide